### DISCUSSION TITLE
No green line

### DISCUSSION THREAD

**Discussion by Mauri740** [dl9rdz/rdz_ttgo_sonde#410] [Q&A] [answered]
hello everyone! While receiving a sonde, the green line is not shown on the map, only the callsign of the receiver is present in the telemetry box. The settings are: 
-SondeHub reporting 1
-SondeHub location reporting 3
what is going on?
Thank you

**Comment by DG0MG** [comment, marked as answer]
If your callsign is displayed in the telemetry box, everything is good! The green lines are not drawn to everyone who receives a frame. Maybe only to the first ones or to those who have the highest reception field strength. This is determined by sondehub, not the rdz-TTGO software.

In the telemetry box you can also press "PLOT", there is a Grafana interface in which you can see when you received the probe and with which field strength. Or you want to know whether your receiver has room for improvement: See how many frames of a probe you have received and to which lowest altitude it can still go.
